spec/nylas/free_busy_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Nylas::FreeBusy do
+  describe "#email" do
+    it "reads and write value for email" do
+      email = "test@example.com"
+
+      result = described_class.new(email: email)
+
+      expect(result.email).to eq(email)
+    end
+  end
+
+  describe "#object" do
+    it "reads and write value for object" do
+      object = "free_busy"
+
+      result = described_class.new(object: object)
+
+      expect(result.object).to eq(object)
+    end
+  end
+
+  describe "#time_slots" do
+    it "reads and write value for time_slots" do
+      time_slots = [
+        {
+          object: "time_slot",
+          status: "busy",
+          start_time: 1_609_439_400,
+          end_time: 1_640_975_400
+        }
+      ]
+
+      result = described_class.new(time_slots: time_slots)
+
+      expect(result.time_slots.to_a.size).to eq(1)
+      time_slot = result.time_slots.last
```

Comment:
Same about iterating over all the objects. Alternative: test that there is only one `time_slot` object in the collection then leave this line. When you have a `.last` and only test that one object, it makes me wonder about the others in the collection.

Not a huge deal at all, just wondering.

Reply:
The array contains only one so `.last` or `.first` is fine I think.